software-examples/kubernetes/azure/openshift/README.md

```diff
@@ -0,0 +1,327 @@
+
+#### Step 2: Grant Storage Account Permissions
+
+1. Navigate to your **Storage Account** in Azure Portal
+2. Go to **Access control (IAM)**
+3. Click **Add** → **Add role assignment**
+4. Select the role **Storage Blob Data Contributor** (or **Storage Blob Data Owner** for full access)
+5. In the **Assign access to** dropdown, select **User, group, or service principal**
+6. Click **Select members** and search for your App Registration name
+
+
+### 4. Configure Helm Values
+
+Edit the `automq-values.yaml` file and customize it for your environment. You need to replace the placeholder values:
+
+**Key Configuration Areas:**
+
+1. **Azure Blob Storage Configuration**:
+   ```yaml
+   global:
+     config: |
+       s3.ops.buckets=1@azblob://<your-ops-bucket>?region=<az-region>&endpoint=<endpoint-url>&authType=static
+       s3.data.buckets=0@azblob://<your-data-bucket>?region=<az-region>&endpoint=<endpoint-url>&authType=static
+       s3.wal.path=0@azblob://<your-data-bucket>?region=<az-region>&endpoint=<endpoint-url>&authType=static
+   ```
+
+2. **Service Principal Credentials** (Static Authentication):
+   ```yaml
+   global:
+     credentials: "static://?accessKey=<client-id>&secretKey=<secret-value>&tenant=<tenant-id>"
+   ```
+
+3. **Resource Limits**: Adjust based on your node capacity (default uses Standard_D4s_v3 recommendations)
+
+4. **StorageClass**: Ensure it matches the StorageClass you created:
+   ```yaml
+   controller:
+     persistence:
+       metadata:
+         storageClass: "automq-disk-azure-premium-ssd"
+   ```
+
+**Important**: Replace all placeholder values marked with `<...>`:
+- `<your-ops-bucket>`: Your Azure Blob Storage operations container name
+- `<your-data-bucket>`: Your Azure Blob Storage data container name
+- `<az-region>`: Your Azure region (e.g., `eastus`)
+- `<endpoint-url>`: Your storage account blob endpoint (e.g., `https://<storage-account>.blob.core.windows.net`)
+- `<client-id>`: Service Principal Client ID (Application ID from App Registration)
+- `<secret-value>`: Service Principal Client Secret (from the client secret you created)
+- `<tenant-id>`: Azure Tenant ID (Directory ID from App Registration)
+
+For more details on available parameters, refer to:
+- [AutoMQ Enterprise Helm Chart Values](https://www.automq.com/docs/automq-cloud/appendix/helm-chart-values-readme)
+- [AutoMQ Enterprise Installation Guide](https://www.automq.com/docs/automq-cloud/appendix/deploy-automq-enterprise-via-helm-chart#install-automq)
+- [AutoMQ Performance Tuning Guide](https://www.automq.com/docs/automq/deployment/performance-tuning-for-broker)
+
+### 5. Install AutoMQ Enterprise
+
+Once your `automq-values.yaml` file is configured, deploy AutoMQ using Helm. Replace `<namespace>` with your desired namespace name:
+
+```bash
+helm install <release-name> oci://automq.azurecr.io/helm/automq-enterprise-chart \
+  -f automq-values.yaml \
+  --version 5.3.2 \
+  --namespace <namespace> \
+  --create-namespace
+```
+
```

Comment:
Please note: After installing Helm, the pod will eventually enter X status due to... To resolve this, follow the instructions in section 5.1. This is a temporary workaround, and we are working on a permanent fix for OpenShift compatibility.